example/components/widgets/CustomNodeWidget.tsx

```diff
@@ -1,61 +1,37 @@
 import * as React from "react";
-import {DiagramEngine, PortWidget} from "@projectstorm/react-diagrams";
+import {DiagramEngine} from "@projectstorm/react-diagrams";
 import {MetaNodeModel} from "../../../.";
-import {Typography} from "@material-ui/core";
+import {Box, Typography} from "@material-ui/core";
+import NodeSelection from "./NodeSelection";
 
 export interface CustomNodeWidgetProps {
     model: MetaNodeModel;
     engine: DiagramEngine;
 }
 
 export class CustomNodeWidget extends React.Component<CustomNodeWidgetProps> {
-
     render() {
-        // @ts-ignore
-        const customNodeStyle = {
-            zIndex: 999999999,
-            border: "solid 2px gray",
-            borderRadius: "5px",
-            width: "250px",
-            height: "200px",
-            display: "flex",
-            alignItems: "flex-start",
-            justifyContent: "space-between",
-            position: "relative",
-            background: this.props.model.getOptions()['color'] || "darkgray",
-            top: this.props.model.getOptions()['position']?.y || 0,
-            left: this.props.model.getOptions()['position']?.x || 0
-        }
-
-        const circlePortStyle = {
-            width: "12px",
-            height: "12px",
-            margin: "2px",
-            borderRadius: "4px",
-            background: "darkgray",
-            cursor: "pointer",
-        }
-
+        const customNodeStyles = {
```

Comment:
@vidhya-metacell why did you choose to use certain local object attributes for styling instead of using a material ui theme that contains the style/styles

Reply:
@zsinnema answered it here https://github.com/MetaCell/PsyNeuLinkView/pull/2#discussion_r929919426